### PR TITLE
fix(graph): Remove inline status messages.

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

File: packages/graph/src/Graph.ts [hunk 1]
```diff
@@ -467,9 +445,9 @@ export class Graph extends SVGZoomWidget {
             .attr("class", "graphEdge")
             .style("opacity", 1e-6)
             .on("click.selectionBag", function (d) {
-                context._selection.click(d, d3Event);
+                context._selection.click(d, event);
```
Comment (jbrundage) [inline packages/graph/src/Graph.ts]: Is this going to work the same way d3Event did?

File: packages/graph/src/Graph.ts [hunk 2]
```diff
@@ -950,6 +930,10 @@ export class Graph extends SVGZoomWidget {
     tooltip;
     tooltipHTML: (_) => string;
     tooltipFormat: (_) => string;
+
+    //  Progess Events ---
+    progress(what: "start" | "end" | "layout-start" | "layout-tick" | "layout-end") {
+    }
```
Comment (jbrundage) [inline packages/graph/src/Graph.ts]: Is the idea to override this method when you'd want to gauge performance?

File: packages/graph/src/Graph.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/graph/src/Graph.ts]: This should not have made it into this PR - good catch.

File: packages/graph/src/Graph.ts (hunk 2 above)
Comment (GordonSmith) [reply packages/graph/src/Graph.ts]: You would add an `on("progess", what=>{})` method to the hosting app.

**Pull request merged**